utils/helpers.ts

```diff
@@ -2,52 +2,34 @@ import os from 'os';
 import fs from 'fs/promises';
 
 // Interfaces
-import { exportedSettings } from './interfaces';
-import { ui_reflected_settings } from './exclusions';
+import { ExportedSettings } from '../utils/types';
+import { uiReflectedSettings } from './exclusions';
 
-let home_dir: String, install_path: String;
-home_dir = os.homedir();
+const homeDir: string = os.homedir();
+let installPath: string;
 
 switch(os.platform()) { 
     case 'linux': { 
-       install_path = 'wp-env';
+       installPath = 'wp-env';
        break; 
     } 
     default: { 
-       install_path = '.wp-env'; 
+       installPath = '.wp-env'; 
```

Comment:
@jeawhanlee  This logic looks linked to `wordpress\env`.
As we don't use it anymore is it still useful?